tests/integration-tests/README.md

```diff
@@ -0,0 +1,532 @@
+
+* BDD encourages collaboration between developers, testers, and non-technical stakeholders (such as business analysts or product owners).
+* In BDD, requirements and specifications are expressed in natural language, often using a specific format called "Given-When-Then." This format structures the description of the expected behavior of a system in a way that is easy to understand.
+* BDD tools, such as Cucumber and Serenity, help translate these specifications into executable tests. These tests serve as living documentation, ensuring that the software behaves as expected based on the agreed-upon specifications.
+
+The Screenplay pattern is used to write the tests. The pattern is described in detail in the
+[Serenity BDD documentation](https://serenity-bdd.github.io/docs/screenplay/screenplay_fundamentals). The main advantages of the Screenplay pattern are:
+
+* The Screenplay Pattern aims to improve the readability, maintainability, and reusability of test automation code.
+* In the Screenplay Pattern, test are written from the perspective of actors who interact with the system. Actors represent the different roles or personas in the system, such as users or components.
+* Each actor has their own abilities (actions they can perform) and interactions (tasks they can carry out). These interactions encapsulate the steps required to achieve a certain goal in the system.
+* The Screenplay Pattern promotes the use of high-level, business-oriented language in test scripts, making them more understandable to non-technical stakeholders.
+
+<p align="center">
+<img src="docs/static/screenplay.png" alt="Screenplay pattern overview" width="750"/><br>
+<em>Pic. 1. Screenplay pattern overview</em>
+</p>
+
+## Project structure
+
+The project structure is represented below:
+```text
+├── main
+│   └── kotlin
+│       └── models -> contains the data models used in the tests (in addition to the PRISM models from client library)
+└── test
+    ├── kotlin
+    │   ├── abilities -> contains the abilities of the actors
+    │   ├── common -> contains the common classes (test constants and helper functions)
+    │   ├── config -> contains the configuration classes (Hoplite)
+    │   ├── features -> contains the features implementation steps
+    │   ├── interactions -> contains the interactions of the actors
+    │   └── runners -> contains the test runners to execute the tests
+    └── resources -> contains the test resources
+        ├── configs -> contains the test configuration files
+        ├── containers -> contains the Docker Compose files to start the test environment
+        └── features -> contains the Gherkin feature files
+```
+
+Here are some rules to follow when writing the tests:
+1. Do not mix steps from different features in the same step definition class. Always create a separate class for each feature.
+2. Do not duplicate configurations, check if the configuration you need already exists in the `test/resources/configs` directory.
```

Comment:
2. Do not duplicate configurations; check if the configuration you need already exists in the `test/resources/configs` directory.